src/MessageHandler/ProcessFileHandler.php

```diff
@@ -110,7 +109,9 @@ public function __invoke(ProcessFile $processFile)
         @$fileStream = fopen($filePath, 'r');
 
         if ($fileStream === false) {
-            $file->setDescription("Unreadable Queued File:" . error_get_last()['message']);
+            $lastErrorMessage = error_get_last()['message'];
+            $this->logger->error(sprintf('Unreadable Queued File: "%s"', $lastErrorMessage, $loggingContext));
+            $file->setDescription('Unreadable Queued File:' . $lastErrorMessage);
```

Comment:
Should we consider setting some sort of max-tries? We'd have to store it in the entity if we did though, but currently if a file is truly unreadable, wouldn't it try forever?

Reply:
This section ends with a return, so no retry.
But otherwise retries are currently limited to (default) 3.